### ISSUE TITLE
"[object response]" error when trying to load videos

### ISSUE THREAD

**Issue by bh08-net** [jellyfin/jellyfin-plugin-intros#20]
I have installed the old plugin, but never got it to work, asked for help, got the new one. After restarting the server, i went to configure it, made a folder for it on my nas (which is on the same device, and locally accessible), and then plopped a video in. I clicked load video, and got the following error:
![image](https://github.com/BrianCArnold/jellyfin-plugin-intros/assets/76684157/8034df96-f1e2-468d-8411-df1018ea0454)

The same happens when i press the clean intros button. The same thing happens even after restarting again. My log files are a bit broken and i cannot read them...

**Comment by GavinGWhiz** [comment]
Jumping in to say the same issue persists with my Jellyfin instance running on a Synology NAS. Just the generic [object Response] popup when I try to load videos.

**Comment by owenashurst** [comment]
I get the same issue. Any fixes?

Edit: I had to manually put in the file path to `/media/intros`. After I clicked off it, it saved, then Load Videos worked.

**Comment by BLACK4585** [comment]
Yep. That's just the error if it can't find any intros. Check your set path. And put some intros in the folder.
